File: src/main/java/com/jozufozu/exnihiloomnia/common/blocks/barrel/BlockBarrelGlass.java
package com.jozufozu.exnihiloomnia.common.blocks.barrel;

import com.jozufozu.exnihiloomnia.common.lib.LibBlocks;
import net.minecraft.block.SoundType;
import net.minecraft.block.material.Material;
import net.minecraft.util.BlockRenderLayer;

public class BlockBarrelGlass extends BlockBarrel
{
    public BlockBarrelGlass()
    {
        super(LibBlocks.GLASS_BARREL, Material.GLASS, SoundType.GLASS);
        this.setHardness(0.7f);
    }
    
    @Override
    public BlockRenderLayer getBlockLayer()
    {
        return BlockRenderLayer.TRANSLUCENT;
    }
}

File: src/main/java/com/jozufozu/exnihiloomnia/common/blocks/barrel/BlockBarrelGlass.kt
package com.jozufozu.exnihiloomnia.common.blocks.barrel

import com.jozufozu.exnihiloomnia.common.lib.LibBlocks

class BlockBarrelGlass : BlockBarrel(LibBlocks.GLASS_BARREL, Material.GLASS, SoundType.GLASS) {
    init {
        this.setHardness(0.7f)
    }

    override fun getBlockLayer() = BlockRenderLayer.TRANSLUCENT
}
